ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/queries/CommonQueries.scala

```diff
@@ -58,16 +57,21 @@ trait CommonQueries extends Queries {
 
   override final def selectStateValuesByKeys(keys: Seq[Key]): Try[immutable.Seq[Option[Value]]] =
     Try {
-      val results = SQL"SELECT key, value FROM #$StateTable WHERE key IN ($keys)"
-        .fold(Map.newBuilder[ByteString, Array[Byte]], ColumnAliaser.empty)((builder, row) =>
-          builder += ByteString.readFrom(row[InputStream]("key")) -> row[Value]("value"))
-        .fold(exceptions => throw exceptions.head, _.result())
-      keys.map(key => results.get(ByteString.copyFrom(key)))(breakOut)
+      val results =
+        SQL"SELECT key, value FROM #$StateTable WHERE key IN (${keys.map(_.toByteArray)})"
```

Comment:
You can make anorm understand `ByteString` directly to avoid cluttering up the query. See https://github.com/digital-asset/daml/pull/4953/files#diff-902c217ea8879e2c95e2294eb8a4113aR111-R119

Reply:
Good idea! I'll let you guys move to using the implicit conversions you wrote when introducing stable offsets.

Reply:
OK -- will add an implicit conversion in `Conversions`.

Reply:
Done.